lib/actions/model.js

```diff
@@ -61,4 +61,25 @@ module.exports = {
       return xhr;
     };
   },
+
+  validate(brainstemKey, attributes, options = {}) {
+    const {
+      validateOptions = {},
+      preValidateAction,
+      postValidateAction,
+    } = options;
+
+    const storageManager = StorageManager.get();
+
+    return (dispatch) => {
+      if (preValidateAction) dispatch(preValidateAction);
+
+      const Model = storageManager.storage(brainstemKey).model;
+      const validationErrors = new Model().validate(attributes, validateOptions);
+
+      if (postValidateAction) { dispatch(postValidateAction(validationErrors)); }
+
+      return validationErrors;
```

Comment:
Wondering whether this should be a promise (to mimic the other actions): valid -> resolve, invalid -> reject?